test/e2e/framework/csidriver/helpers.go

```diff
@@ -0,0 +1,151 @@
+/*
```

Comment:
nit: Just `csidriver.go` would be a better name for this file IMHO. I always try to avoid `helpers` or `utils` etc unless there really is nothing more descriptive.